### PR TITLE
US126718 -- Style Lint

### PR DESCRIPTION
**Pull request by mpharoah-d2l**
d2l-op-adds d2l-op-style d2l-op-linting d2l-op-and d2l-op-fixes d2l-op-problems

### CODE REVIEW

File: d2l-program-outcomes-picker.js [hunk 1]
```diff
@@ -69,30 +69,23 @@ class ProgramOutcomesPicker extends  LocalizeMixin(LitElement) {
...
 				padding: 11px 40px;
 			}
 			
-			.button-spacer {
+			.d2l-op-button-spacer {
 				display: inline-block;
 				width: 13px;
 			}
 			
-			.button-list {
```
Comment (mpharoah-d2l) [inline d2l-program-outcomes-picker.js:82]: This CSS class was unused

**Comment by devpow112** [comment]
I'd like to wait on this to see the results of whatever conversation happens with https://github.com/BrightspaceUI/stylelint-config/pull/5. I don't want us to just follow a standard if we don't agree with it and I'd rather not have our config diverge from the "standard" config if we can avoid it. I'm guessing the ones were you were worrying about collision if we used just the `d2l-` prefix was where we include external styles in our own like this https://github.com/Brightspace/program-outcomes-picker/blob/a9f8528cd858a6b0657c7183a18914873e2e4424/d2l-asn-outcomes-picker.js#L150. Just want to make sure I understand the concern about collisions you mentioned.

**Comment by mpharoah-d2l** [comment]
Yeah, like changing `.body` to `.d2l-body` would probably result in the text styling being pulled in too

**Comment by devpow112** [comment]
Thanks for your patience with this.

**Comment by devpow112** [comment]
We've discussed this but due to timing did not come to a conclusion with team Gaudi. For now let's proceed with these changes as is and we will revisit this once we can have the conversation in full.

**Review by devpow112 — APPROVED** [review]

**Pull request merged**